src/scss/index.scss

```diff
@@ -425,23 +425,36 @@ table.field-list {
   color: #e83e8c;
 }
 
-/* Social media buttons */
+/* Icon Links */
 
-i {
-  &.fab {
-    vertical-align: middle;
-    font-style: normal;
-    font-size: 1.5rem;
-    line-height: 1.25;
+#navbar-icon-links {
+  i {
+    &.fa,
+    &.fab,
+    &.far,
+    &.fas{
+      vertical-align: middle;
+      font-style: normal;
+      font-size: 1.5rem;
+      line-height: 1.25;
+    }
   }
+}
 
+/* Social media buttons */
+
+i {
```

Comment:
Should we include this in the `navbar-icon-links` selector above, so those colors are only used there?

Reply:
limited in d616236d!